examples/eng_wasm_contracts/contract_with_eth_calls/src/lib.rs

```diff
@@ -52,5 +58,23 @@ impl ContractInterface for Contract {
         let key = "code";
         eprint!("{}", a);
         write_state!(key => &a);
+        let sum: u64 = 12;
+        write_state!("sum" => sum);
     }
-}
+
+    fn sum_and_call(x: U256, y: U256, eth_addr: H160) -> U256 {
+        let sum: u64 = x.as_u64() + y.as_u64();
+        write_state!("code" => sum);
+        let mut eth_addr_str_0x: String = "0x".to_string();
+        let eth_addr_str: String = eth_addr.to_hex();
+        eth_addr_str_0x.push_str(&eth_addr_str);
+        let c = EthContract::new(&eth_addr_str_0x);
+        c.record(sum.into());
+        sum.into()
+    }
+
+    fn get_last_sum() -> U256 {
+        let sum: u64 = read_state!("sum").unwrap_or_default();
+        sum.into()
+    }
+}
```

Comment:
Please add a newline at the end of files. Git complains if you omit it (the red arrow here)